frontend/src/v5/ui/components/shared/appBar/appBar.component.tsx

```diff
@@ -22,24 +22,30 @@ import LogoIcon from '@assets/icons/logo.svg';
 import IntercomIcon from '@assets/icons/intercom.svg';
 import NotificationsIcon from '@assets/icons/notifications.svg';
 import { CircleButton } from '@/v5/ui/controls/circleButton';
-import { AvatarButton } from '@/v5/ui/controls/avatarButton';
+import { CurrentUserHooksSelectors } from '@/v5/services/selectorsHooks/currentUserSelectors.hooks';
+import { UserMenu } from '../userMenu';
 import { Items } from './appBar.styles';
 import { Breadcrumbs } from '../breadcrumbs';
 
-export const AppBar = (): JSX.Element => (
-	<MuiAppBar position="static">
-		<Items>
-			<LogoIcon />
-			<Breadcrumbs />
-		</Items>
-		<Items>
-			<CircleButton variant="contrast" aria-label="intercom">
-				<IntercomIcon />
-			</CircleButton>
-			<CircleButton variant="contrast" aria-label="notifications">
-				<NotificationsIcon />
-			</CircleButton>
-			<AvatarButton> GH </AvatarButton>
-		</Items>
-	</MuiAppBar>
-);
+export const AppBar = (): JSX.Element => {
+	const user = CurrentUserHooksSelectors.selectCurrentUser();
+	return (
+		<>
```

Comment:
is it necessary to add these `</>`  or that causes a problem?

Reply:
Removed